OMCompiler/SimulationRuntime/c/simulation/solver/initialization/initialization.c

```diff
@@ -277,9 +278,10 @@ static int symbolic_initialization(DATA *data, threadData_t *threadData)
       sprintf(buffer, "%s_equidistant_global_homotopy.csv", mData->modelFilePrefix);
       infoStreamPrint(LOG_INIT, 0, "The homotopy path will be exported to %s.", buffer);
       pFile = omc_fopen(buffer, "wt");
-      fprintf(pFile, "\"sep=%s\"\n%s", sep, "\"lambda\"");
-      for(i=0; i<mData->nVariablesReal; ++i)
+      fprintf(pFile, "\"lamda\"");
```

Comment:
Shouldn't this say lambda?

Reply:
Yes, ouf course. Thanks for finding the tipo.